packages/apps/lightning-ui-docs/.storybook/theme.js

```diff
@@ -16,14 +16,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { create } from '@storybook/theming/create';
+import { create, themes } from '@storybook/theming/create';
 import brand from './brand.png';
-import { version } from '@lightningjs/ui-components/package.json';
+import { version } from '../../../@lightningjs/ui-components/package.json'; // TODO: follow up on this temp fix to get the import to work
```

Comment:
still needed?

Reply:
is this still needed? what's the context?

Reply:
Error: `The path "./package.json" is not exported by package "@lightningjs/ui-components":`
If we want to export the file some how then it may work but we also may run into the other issue with .mjs & Import
see comment on ThemeDownload.js